action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -466,12 +466,16 @@ func (stateDB *StateDBAdapter) AddLog(evmLog *types.Log) {
 		}
 		from, _ := address.FromBytes(topics[1][12:])
 		to, _ := address.FromBytes(topics[2][12:])
-		stateDB.transactionLogs = append(stateDB.transactionLogs, &action.TransactionLog{
-			Type:      iotextypes.TransactionLogType_IN_CONTRACT_TRANSFER,
-			Sender:    from.String(),
-			Recipient: to.String(),
-			Amount:    new(big.Int).SetBytes(evmLog.Data),
-		})
+		amount := new(big.Int).SetBytes(evmLog.Data)
+		//ignore invalid log
+		if amount.BitLen() != 0 {
+			stateDB.transactionLogs = append(stateDB.transactionLogs, &action.TransactionLog{
```

Comment:
move `from, to` inside this if{}

Reply:
also, can you check why test fail?
if this will cause block hash to change, need a new hard-fork

Reply:
The failure of CI test might be caused by the code in `integrity_test.go`. I will look into it. 
> FAIL	github.com/iotexproject/iotex-core/blockchain/integrity	0.565s
FAIL
Exited with code exit status 1